charts/filsnap/values.yaml

```diff
@@ -0,0 +1,127 @@
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+commonLabels: {}
+
+image:
+  repository: ghcr.io/travisperson/filsnap
+  pullPolicy: IfNotPresent
+  tag: ""
+
+snapshots:
+  # A list of multiaddrs of nodes to use for creating snapshots
+  nodes: []
+
+  # Cron schedule to run the snapshot creation process on, the frequency
+  # of this schedule should match that of the `interval` value.
+  schedule: "0 */2 * * *"
+
+  # Epoch interval used to calculate the correct snapshot height when a snapshot
+  # job starts
+  interval: "240"
+
+  # Number of epochs that must exist after the calculated snapshot height before
+  # the snapshot is started
+  confidence: "30"
+
+  jwtTokens:
+    # Name of secret containing the jwt token. All nodes should share a common
+    # jwt secret and token. The token must have read and admin flags set.
+    secretName: jwt-token
+    items:
+    - key: jwt-all-privs-token
+      path: token
+
+  logging:
+    level: "info"
+    # List of loggers
+    named: {}
+    # - logger: "filsnap/operator"
+    #   level: "error"
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  uploads:
+    # When disabled snapshots will be discarded
+    enabled: false
+
+    # Name of bucket to upload snapshots into
+    bucket: ""
+
+    # Url endpoint to upload snapshots into
+    endpoint: ""
+
+    # When set a secret will be created. Further updates to the chart will lookup
+    # the values so that that these values do not need to be passed in multiple times
+    accessKey: ""
+    secretKey: ""
```

Comment:
Probably also want to support an external secret, but probably will add that when I run into the need for it.